### ISSUE TITLE
add rfkill kill switch support

### ISSUE THREAD

**Issue by jlo62** [ErikReider/SwayOSD#94]
Hi, it would be great to have a osd when I toggle wifi/bluetooth.

This would simply need to run:
```
rfkill block [identifier]
rfkill unblock [identifier]
rfkill toggle [identifier]
```
[identifier] can be àll`, `wlan`, `bluetooth` or a number or string associated with a specific device.\
[identifier] would be added as a command line argument.

As far as I know rfkill should be available for all users by default, so this wouldn't need root.

btw - great project, been looking for something like this for a while!

\- Jan Luca

**Comment by ErikReider** [comment]
Duplicate of #77
